documentation/latest/intro/overview.md

```diff
@@ -3,34 +3,43 @@ layout: documentation
 title: What is flotta
 ---
 
-## Phase 1
-
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
-Ipsum has been the industry's standard dummy text ever since the 1500s, when an
-unknown printer took a galley of type and scrambled it to make a type specimen
-book. It has survived not only five centuries, but also the leap into electronic
-typesetting, remaining essentially unchanged. It was popularised in the 1960s
-with the release of Letraset sheets containing Lorem Ipsum passages, and more
-recently with desktop publishing software like Aldus PageMaker including
-versions of Lorem Ipsum.
-
-## Phase 2
-
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
-Ipsum has been the industry's standard dummy text ever since the 1500s, when an
-unknown printer took a galley of type and scrambled it to make a type specimen
-book. It has survived not only five centuries, but also the leap into electronic
-typesetting, remaining essentially unchanged. It was popularised in the 1960s
-with the release of Letraset sheets containing Lorem Ipsum passages, and more
-recently with desktop publishing software like Aldus PageMaker including
-versions of Lorem Ipsum.
-## Phase 3
-
-Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem
-Ipsum has been the industry's standard dummy text ever since the 1500s, when an
-unknown printer took a galley of type and scrambled it to make a type specimen
-book. It has survived not only five centuries, but also the leap into electronic
-typesetting, remaining essentially unchanged. It was popularised in the 1960s
-with the release of Letraset sheets containing Lorem Ipsum passages, and more
-recently with desktop publishing software like Aldus PageMaker including
-versions of Lorem Ipsum.
+## Fleet management for edgedevices
+
+Flotta adds a way to manage all your fleet of devices from a single point, in
+this case, using Kubernetes CRD to handle all kinds of device configurations.
+
+Each Edgedevice can register, pull configuration using the Fleet Management API
+and keep syncing the data using a secure MTLS protocol between the device and
+the Fleet Management API.
+
+At the same time, you can schedule a container workload to run in the device
```

Comment:
Not sure: s/At the same time, you can schedule / It is alsoo possible to schedule/